tools/check-git-version.mjs

```diff
@@ -0,0 +1,22 @@
+import semver from 'semver';
+import shell from 'shelljs';
+import simpleGit from 'simple-git';
+
+const GIT_MINIMUM_VERSION = '2.33.0';
+const git = simpleGit();
+// eslint-disable-next-line @typescript-eslint/no-floating-promises
+(async () => {
+  try {
+    const regex = /\d+\.\d+\.\d+/;
+    const stdout = await git.raw('--version');
+    const [gitVersion] = regex.exec(stdout);
+    if (semver.lt(gitVersion, GIT_MINIMUM_VERSION)) {
+      throw new Error(`Minimum Git version ${GIT_MINIMUM_VERSION} is required`);
```

Comment:
```suggestion
      if (process.env.CI) {
        shell.echo(`::error ::Minimum Git version ${GIT_MINIMUM_VERSION} is required`);
      } else {
        throw new Error(`Minimum Git version ${GIT_MINIMUM_VERSION} is required`);
      }
```

do not fail on CI

Reply:
https://docs.github.com/en/actions/reference/workflow-commands-for-github-actions#setting-an-error-message